public/editor/scripts/editor/js/fc/publisher.js

```diff
@@ -19,8 +20,7 @@ define(function(require) {
       .on("click", publish);
   }
 
-  function Publisher(options) {
-    this.fsync = options.sync;
+  function Publisher() {
```

Comment:
I don't think we need the `dialog.trackSyncChanges` property anymore; we don't use it anywhere.